src/plugins/code-block.tsx

```diff
@@ -1,61 +1,61 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const prism = require("prismjs");
+const prism = require('prismjs');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const loadLanguages = require("prismjs/components/");
+const loadLanguages = require('prismjs/components/');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const Html5Entities = require("html-entities");
+const Html5Entities = require('html-entities');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const unified = require("unified");
+const unified = import('unified').then((mod) => mod.default);
```

Comment:
@timngyn we had to make this change because `unified` has updated to be an ESM module. We should make sure this works with the static site build as well as ensure we haven't missed any other deps that need to be switched from `require` to `import`.